plugins/foreman_templates/6.0/index.md

```diff
@@ -120,120 +120,98 @@ sudo -u foreman git config --global http.sslCAPath /path/to/ca/certs
 
 ```sh
-foreman-rake templates:import repo=https://github.com/theforeman/community-templates
-```
-
-The example above would clone the repository to some temp directory. If `branch` was specified, it switches to this branch. Then if `dirname` was provided, it will "cd" into it. Finally it searches for all files in this directory and subdirectories that has `'.erb` suffix. Then the template is parsed and imported (see below for more details)
-
-```sh
-foreman-rake templates:import repo=/var/lib/my_templates
+curl -H "Accept:application/json,version=2" -H "Content-Type:application/json" -u user1:changeme -k https://foreman.example.com/api/v2/templates/import -X POST -d "{\"repo\":\"/another/repo\"}"
 ```
 
-Compared to previous example, this uses local directory. Therefore not git clone or branch checkout is done. If `dirname` is specified, the task goes into this subdirectory and searches for all files with '.erb' extension. Then the same parsing and importing process follows,
-
-The parsing process reads the content of the file. It expects the content of the template including some template metadata. The metadata are an ERB comment on top of the file, for illustration see some template in [community-templates repository](https://github.com/theforeman/community-templates/blob/develop/provisioning_templates/provision/kickstart_default.erb). It's in yaml format and specifies template name, kind, compatible operating systems, organizations, locations and possibly other. The template gets it's name based on information from metadata. Before the template is saved in database, `prefix` setting is applied to it's name. Then the `filter` regular expression is matched against the prefixed name. If the `filter` matches, templates is saved. If `negate` is set to `true`, the template would be saved only if `filter` regular expression didn't match the template name. If the template with a given name exists in database already, it's content is updated.
-
-The result of the whole import is printed on the STDOUT. The output contains list of created/updated templates as well as applied diffs.
-
-A few more useful examples follow
-
-```sh
-# prefix all templates with "[comunity] " string to avoid overriding existing templates
-foreman-rake templates:import prefix='[community] '
-
-# get the latest templates (might be compatible with only Foreman nightly)
-foreman-rake templates:import repo=https://github.com/theforeman/community-templates branch=develop
-
-tree /tmp/my-repo/
-/tmp/my-repo/
-└── large-ptable.erb
-
-0 directories, 1 file
-
-# this will create a template named "[community] large-ptable"
-foreman-rake templates:import repo=/tmp/my-repo/ prefix='[community] '
-
+a ruby description of a JSON you get back follows
```

Comment:
Why use a Ruby syntax to describe JSON? Even the syntax for the snippet has json syntax highlighting. All it takes to turn it to a proper json is `s/=>/:/g` and `s/nil/null/g`.